### DISCUSSION TITLE
[selenium-webdriver] Could not find a declaration file for module 'selenium-webdriver/lib/error'

### DISCUSSION THREAD

**Discussion by MaxNanasy** [DefinitelyTyped/DefinitelyTyped#60936] [Issues with a @types package]
With the code line:

```
import { NoSuchElementError } from 'selenium-webdriver/lib/error';
```

Visual Studio gives the warning message:

```
Could not find a declaration file for module 'selenium-webdriver/lib/error'. '<...>/node_modules/selenium-webdriver/lib/error.js' implicitly has an 'any' type.
  If the 'selenium-webdriver' package actually exposes this module, consider sending a pull request to amend 'https://github.com/DefinitelyTyped/DefinitelyTyped/tree/master/types/selenium-webdriver'ts(7016)
```

The code works at runtime though

Issue happens with @types/selenium-webdriver version 4.0.18 or 4.1.1

[a comment by an automated account is not shown]

**Comment by SupernaviX** [comment]
It looks like the errors have been defined in a [namespace inside of index.d.ts](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/selenium-webdriver/index.d.ts#L44). I _think_ that moving that namespace into a lib/error.d.ts file would fix the issue.

Mind submitting a PR?
